client/components/NavBar.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Route, Switch, BrowserRouter, Redirect, Link } from "react-router-dom"

```

Comment:
We can remove `BrowserRouter` from the import statement since it's not being used